clusters/build-clusters/01_cluster/openshift-machine-api/admin_build01-9hdwj-worker-us-east-1a-m5d4x_machine_set.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineSet
+metadata:
+  labels:
+    machine.openshift.io/cluster-api-cluster: build01-9hdwj
+  name: build01-9hdwj-worker-us-east-1a-m5d4x
+  namespace: openshift-machine-api
+spec:
+  selector:
+    matchLabels:
+      machine.openshift.io/cluster-api-cluster: build01-9hdwj
+      machine.openshift.io/cluster-api-machineset: build01-9hdwj-worker-us-east-1a-m5d4x
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        machine.openshift.io/cluster-api-cluster: build01-9hdwj
+        machine.openshift.io/cluster-api-machine-role: worker
+        machine.openshift.io/cluster-api-machine-type: worker
+        machine.openshift.io/cluster-api-machineset: build01-9hdwj-worker-us-east-1a-m5d4x
+    spec:
+      metadata:
+        creationTimestamp: null
+      providerSpec:
+        value:
+          ami:
+            id: ami-06f85a7940faa3217
+          apiVersion: awsproviderconfig.openshift.io/v1beta1
+          blockDevices:
+          - ebs:
+              iops: 0
```

Comment:
Do we really want this? Or just omit it?

Reply:
It is in the doc.
https://docs.openshift.com/container-platform/4.1/machine_management/creating-machineset.html#machineset-yaml_creating-machineset

Removing ...

Reply:
Done.